src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+    private final int number = 100;

```

Comment:
Please use capital letters when create constants.
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final int NUMBER_LIMIT = 100;

```

Reply:
After adding static i get this error
Lottery.java:8:5: Static variable definition in wrong order. [DeclarationOrder]